arbnode/schema.go

```diff
@@ -4,15 +4,16 @@
 package arbnode
 
 var (
-	messagePrefix                []byte = []byte("m") // maps a message sequence number to a message
-	blockHashInputFeedPrefix     []byte = []byte("b") // maps a message sequence number to a block hash received through the input feed
-	blockMetadataInputFeedPrefix []byte = []byte("t") // maps a message sequence number to a blockMetaData byte array received through the input feed
-	messageResultPrefix          []byte = []byte("r") // maps a message sequence number to a message result
-	legacyDelayedMessagePrefix   []byte = []byte("d") // maps a delayed sequence number to an accumulator and a message as serialized on L1
-	rlpDelayedMessagePrefix      []byte = []byte("e") // maps a delayed sequence number to an accumulator and an RLP encoded message
-	parentChainBlockNumberPrefix []byte = []byte("p") // maps a delayed sequence number to a parent chain block number
-	sequencerBatchMetaPrefix     []byte = []byte("s") // maps a batch sequence number to BatchMetadata
-	delayedSequencedPrefix       []byte = []byte("a") // maps a delayed message count to the first sequencer batch sequence number with this delayed count
+	messagePrefix                       []byte = []byte("m")  // maps a message sequence number to a message
+	blockHashInputFeedPrefix            []byte = []byte("b")  // maps a message sequence number to a block hash received through the input feed
+	blockMetadataInputFeedPrefix        []byte = []byte("t")  // maps a message sequence number to a blockMetaData byte array received through the input feed
+	missingBlockMetadataInputFeedPrefix []byte = []byte("mt") // maps a message sequence number whose blockMetaData byte array is missing to nil
```

Comment:
Oh shoot, yes it breaks iterating over messages! 
Since its tracking missing stuff, I propose lets go with "x".

Maybe to even future proof this- what if we go with "xt" meaning we are tracking missing blockmetadata, for example "xb" would mean missing blockHash etc...?